x-pack/plugin/fleet/src/main/java/org/elasticsearch/xpack/fleet/rest/RestGetGlobalCheckpointsAction.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.fleet.rest;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.unit.TimeValue;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.fleet.action.GetGlobalCheckpointsAction;
+
+import java.util.List;
+
+import static org.elasticsearch.rest.RestRequest.Method.GET;
+
+public class RestGetGlobalCheckpointsAction extends BaseRestHandler {
+
+    @Override
+    public String getName() {
+        return "fleet_get_global_checkpoints";
+    }
+
+    @Override
+    public List<Route> routes() {
+        return List.of(new Route(GET, "/{index}/_fleet/global_checkpoints"));
```

Comment:
I wonder if we have a convention of plugins using completely separate namespaces or not? This is fine by me, but could also be `_fleet/{index}/...`. At least for data stream delete we chose that, but this is also somewhat different in that it concerns indices. Will try to look for examples later...

Reply:
I am fine with whatever. I followed the ccr example for this PR.

Reply:
Ok by me to take this approach, looks like we do so on several occassions.